src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+// import { User } from './types/User';
+
+// function getTodos() {
+//   return fetch('http://localhost:5173/api/todos.json').then(response => {
+//     return response.json();
+//   });
+// }
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoaiding] = useState(true);
```

Comment:
There is a typo in the state variable `setLoaiding`. It should be `setLoading` to match the variable name `loading`.